security/crunch/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+compiler.whitelist \
+                clang
```

Comment:
We don't generally whitelist a single compiler. I think this statement should be removed.

Reply:
@mf2k Do I not need to force it to use clang? Or `CC=${configure.cc}` enough to ensure a good compiler is used?

Reply:
`${configure.cc} `is the correct compiler as determined by port, so that's what you need to use.